src/ContractCallDataEncoder.cpp

```diff
@@ -382,6 +389,8 @@ QVariant ContractCallDataEncoder::decodeRawArray(SolidityType const& _type, byte
 		value.populate(&rawParam);
```

Comment:
If I see that correctly, then two lines above you create a bytesConstRef that can span 32 bytes beyond the end of the byte array.